api/include/opentelemetry/semconv/incubating/system_metrics.h

```diff
@@ -277,12 +277,13 @@ static inline nostd::shared_ptr<metrics::ObservableInstrument> CreateAsyncDouble
  * Time disk spent activated
  * <p>
  * The real elapsed time ("wall clock") used in the I/O path (time from operations running in
- * parallel are not counted). Measured as: <ul> <li>Linux: Field 13 from <a
- * href="https://www.kernel.org/doc/Documentation/ABI/testing/procfs-diskstats">procfs-diskstats</a></li>
- *   <li>Windows: The complement of
- * <a
- * href="https://learn.microsoft.com/archive/blogs/askcore/windows-performance-monitor-disk-counters-explained#windows-performance-monitor-disk-counters-explained">"Disk%
- * Idle Time"</a> performance counter: @code uptime * (100 - "Disk\% Idle Time") / 100 @endcode</li>
+ * parallel are not counted). Measured as: <ul> 
+ *  <li>Linux: Field 13 from 
+ *    <a href="https://www.kernel.org/doc/Documentation/ABI/testing/procfs-diskstats">procfs-diskstats</a>
+ *  </li>
+ *  <li>Windows: The complement of 
+ *    <a href="https://learn.microsoft.com/archive/blogs/askcore/ windows-performance-monitor-disk-counters-explained#windows-performance-monitor-disk-counters-explained">"Disk% Idle Time"</a> 
+ * performance counter: @code uptime * (100 - "Disk\% Idle Time") / 100 @endcode</li>
```

Comment:
Quick Question: Is this auto-formatted? 
If **not**, I think `<ul>`  should start from a new line.